app/src/main/java/com/kickstarter/services/ApiClient.kt

```diff
@@ -0,0 +1,72 @@
+package com.kickstarter.services
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import com.google.gson.Gson
+import com.kickstarter.libs.Config
+import com.kickstarter.services.apiresponses.ErrorEnvelope
+import retrofit2.Response
+import retrofit2.http.GET
+import java.io.EOFException
+import kotlin.coroutines.cancellation.CancellationException
+
+interface ApiService {
+    @GET("/v1/app/android/config")
+    suspend fun getConfigResponse(): Response<Config>
```

Comment:
@leighdouglas let me know if any other endpoint is required for the SplashScreen initialization project and we can add it right away